sys-devel/clang-common/clang-common-19.0.0_pre20240203.ebuild

```diff
@@ -174,11 +174,17 @@ src_install() {
 		-include "${EPREFIX}/usr/include/gentoo/fortify.h"
 	EOF
 
-	newins - gentoo-hardened-ld.cfg <<-EOF
-		# Some of these options are added unconditionally, regardless of
-		# USE=hardened, for parity with sys-devel/gcc.
-		-Wl,-z,relro
-	EOF
+	if use kernel_Darwin; then
+		newins - gentoo-hardened-ld.cfg <<-EOF
+			# There was -Wl,-z,relro here, but it's not supported on Mac
```

Comment:
ld64 supports
>
>     -bind_at_load
>            Sets a bit in the mach header of the resulting binary which tells dyld to bind all symbols when the binary >is loaded, rather than lazily.


>     -read_only_stubs
>             [i386 only] Makes the __IMPORT segment of a final linked images read-only.  This option makes a >program slightly more secure in that the JMP instructions in the i386 fast
>             stubs cannot be easily overwritten by malicious code.  The downside is the dyld must use mprotect() to >temporarily make the segment writable while it is binding the
>             stubs.

I haven't tested either so please just add a comment mentioning those two options for further investigation, then LGTM>

Reply:
Done